src/Repository/AdherentRepository.php

```diff
@@ -553,6 +553,29 @@ public function countSupervisorsByGenderForReferent(Adherent $referent): array
         return $this->formatCount($result);
     }
 
+    public function countMembersByGenderForReferent(Adherent $referent): array
+    {
+        if (!$referent->isReferent()) {
+            throw new \InvalidArgumentException('Adherent must be a referent.');
+        }
+
+        $result = $this->createQueryBuilder('adherent', 'adherent.gender')
+            ->select('adherent.gender, COUNT(DISTINCT adherent) AS count')
+            ->join('adherent.memberships', 'membership')
+            ->join(Committee::class, 'committee', Join::WITH, 'committee.uuid = membership.committeeUuid')
+            ->innerJoin('committee.referentTags', 'tag')
+            ->where('tag.id IN (:tags)')
```

Comment:
`committee.referentTags IN (...)`?

Reply:
It does not work and throws en error: `Error: Invalid PathExpression. StateFieldPathExpression or SingleValuedAssociationField expected.`